h/models/annotation.py

```diff
@@ -114,6 +132,10 @@ class Annotation(Base):
                             sa.ForeignKey('document.id'),
                             nullable=False)
 
+    motivations = sa.Column(MutableList.as_mutable(
+        pg.ARRAY(sa.UnicodeText, zero_indexes=True)
+     ), default=list)
```

Comment:
It seems to produce the same result in the DB, but I'm not sure why the `motivations` column in the model class looks different than the one in the migration? The model class has `MutableList.as_mutable` but the migration doesn't.

Reply:
Should there be a `server_default` here as well as `default`?